airflow/utils/state.py

```diff
@@ -41,6 +41,21 @@ class State(object):
     UPSTREAM_FAILED = "upstream_failed"
     SKIPPED = "skipped"
 
+    task_states = (
+        SUCCESS,
+        RUNNING,
+        FAILED,
+        UPSTREAM_FAILED,
+        UP_FOR_RETRY,
+        QUEUED,
+    )
```

Comment:
Yes, I believe that was because to the GUI or tasks table didn't
distinguish scheduled tasks vs running tasks, or something like that.

On Wed, Sep 7, 2016 at 10:23 PM, bolkedebruin <notifications@github.com>
wrote:

> In airflow/utils/state.py
> <https://github.com/apache/incubator-airflow/pull/1662#discussion_r77947508>
> :
>
> > @@ -41,6 +41,21 @@ class State(object):
> >      UPSTREAM_FAILED = "upstream_failed"
> >      SKIPPED = "skipped"
> >
> > +    task_states = (
> > +        SUCCESS,
> > +        RUNNING,
> > +        FAILED,
> > +        UPSTREAM_FAILED,
> > +        UP_FOR_RETRY,
> > +        QUEUED,
> > +    )
>
> Ok. Understood. Still it doesnt cover all task states (scheduled). Maybe
> that should be added
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-airflow/pull/1662/files/f734d8a876b6a31abde57cedf5e44c69031f82b1#r77947508>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGYMpqhonQsqsBZXzBOaeV0JuL5WpAsAks5qn5vrgaJpZM4JLnBS>
> .
>



-- 
Thank you,

Norman Mu